nightly-benchmark/run-dgx-benchmark.sh

```diff
@@ -19,6 +19,13 @@ ENV="$TODAY-nightly-0.17"
 conda activate $ENV
 which python
 
+echo "Cythonize Distributed"
+pushd "${CONDA_PREFIX}/lib/python3.8/site-packages/"
+cythonize -i \
+	"distributed/protocol/serialize.py"
+	"distributed/scheduler.py"
```

Comment:
We haven't made any changes here yet, but it seems like a good idea to queue this up so we are ready once we do.